.gitignore

```diff
@@ -1,3 +1,7 @@
 .idea/
 target/
 *.iml
+/.settings/
```

Comment:
@amihaiemil 
> Is there something in teamed.io's projects that cannot or shouldn't be done in Eclipse?

No, you can code even in VIM, but you'll most probably encounter poor support for Eclipse in projects, like e.g. missing `.gitignore`